api/gateway.proto

```diff
@@ -158,8 +158,8 @@ message UpdateGatewayRequest {
 
 message CreateGatewayAPIKeyRequest {
   GatewayIdentifiers gateway_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
-  string name = 2;
-  repeated Right rights = 3;
+  string name = 2 [(validate.rules).string.max_len = 50];
+  repeated Right rights = 3 [(validate.rules).repeated.items.enum.defined_only = true];
```

Comment:
I could not figure out to do this - please check if there are other repeated enums lacking validations.

Reply:
Checked. I think I got all repeated enums in this PR